tools/workflow/docker-version-compare.sh

```diff
@@ -98,3 +98,4 @@ echo "::set-output name=minor-update::${MINOR_UPDATE}"
 echo "::set-output name=patch-update::${PATCH_UPDATE}"
 echo "::set-output name=any-update::${ANY_UPDATE}"
 echo "::set-output name=same-version::${SAME_VERSION}"
+echo "::set-output name=latest-version::${LATEST_VERSION}"
```

Comment:
Oh this isn't going to work. The latest version is considered v0.16.0. It's just getting the largest version. It isn't checking the digest for both tags.

Reply:
Yes. You got it right though. Will try to change this in the  next commit.